02-electric/electric-section-one/electric-section-one.css

```diff
@@ -1,45 +1,114 @@
+.electric-section-one-container {
+  padding: 15px;
+}
 
-.electric-section-one-container{
-padding: 15px;
+.electric-section-one-container > * {
```

Comment:
please remove this selector 
and add border: solid 1px black; into the .electric-section-one-subcontainer

Reply:
done